Dokan/Dokan/Scenes/Registration/RegisterViewController.xib

```diff
@@ -0,0 +1,133 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
- Constraints should be multiples of 4 as agreed. 
- I prefer using stack views for a minimum constraints and easier adoption to SwiftUI. This might be acceptable for the current phase, but I do expect you to cooperate and implement it using StackView.